[TO MERGE AGAINST 4.9 only] W-16722425: Updating section about modifying wrapper properties (#2825)
mulesoft/docs-mule-runtime-3: modules/ROOT/pages/runtime-installation-task.adoc

== Configure Properties in the wrapper.conf File

> After installing Mule as a service/daemon, you can modify the properties defined in the `wrapper.conf` file.

Reviewer: ```suggestion
After installing Mule as a service or as a daemon, you can modify the properties defined in the `wrapper.conf` file.
```

Author: I used the slash because they are two different terms for something very similar (typically `service` is used in Windows environments and `daemon` is used for Unix environments). They both refer to something that is constantly running in the background.
It is also referenced like that [here](https://github.com/mulesoft/docs-mule-runtime/blob/926a02af3879c231aff7547727426832a6c31da6/modules/ROOT/pages/runtime-installation-task.adoc?plain=1#L69) and [here](https://github.com/mulesoft/docs-mule-runtime/blob/926a02af3879c231aff7547727426832a6c31da6/modules/ROOT/pages/runtime-installation-task.adoc?plain=1#L109).

> Consult https://wrapper.tanukisoftware.com/doc/english/properties.html for details about what you can do with the configuration file.

Reviewer: ```suggestion
For details on what you can do with the configuration file, see https://wrapper.tanukisoftware.com/doc/english/properties.html.
```

> After making modifications to the `wrapper.conf` file, restart Mule to allow the changes to take effect.

Reviewer: ```suggestion
After modifying the `wrapper.conf` file, restart your Mule instance to enable the changes.
```

NOTE: When adding new properties, make sure there are no other properties using the same sequence number already (you do not need to worry about the ones in `wrapper-additional.conf`, those will be adjusted upon restart).

> NOTE: Do not modify the `wrapper-additional.conf` file. That file is automatically generated by Mule.

Reviewer: ```suggestion
[NOTE]
====
When you add new properties, ensure that no other properties already use the same sequence number (don't worry about the ones in `wrapper-additional.conf`, as Mule adjusts those upon restart). +

Don't modify the `wrapper-additional.conf` file, as Mule automatically generates this file.
====
```

== Configure Properties in the wrapper.conf File

After installing Mule as a service or as a daemon, you can modify the properties defined in the `wrapper.conf` file.

> For details on what you can do with the configuration file, see https://wrapper.tanukisoftware.com/doc/english/properties.html.

Reviewer: ```suggestion
For details about what you can do with the configuration file, see https://wrapper.tanukisoftware.com/doc/english/properties.html[Configuration Property Overview].
```

After modifying the `wrapper.conf` file, restart your Mule instance to enable the changes.

[NOTE]
====

> When you add new properties, ensure that no other properties already use the same sequence number (don't worry about the ones in `wrapper-additional.conf`, as Mule adjusts those upon restart). +

Reviewer: ```suggestion
When you add new properties, ensure that no other properties already use the same sequence number (don't worry about the ones in `wrapper-additional.conf`, as Mule adjusts those upon restart).
```
